<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReadingListDetailsWindow</class>
 <widget class="QWidget" name="ReadingListDetailsWindow">
  <property name="geometry">
   <rect><x>0</x><y>0</y><width>800</width><height>600</height></rect>
  </property>
  <property name="minimumSize">
   <size><width>800</width><height>600</height></size>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="text">
      <string>ðŸ“š Reading List: [List Name]</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <bold>true</bold>
      </font>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="bookTable"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <widget class="QPushButton" name="addBookButton">
       <property name="text">
        <string>Add a Book</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removeBookButton">
       <property name="text">
        <string>Remove Book</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="showGoalsButton">
       <property name="text">
        <string>Show Related Goals</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteListButton">
       <property name="text">
        <string>Delete This List</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>